PyInstaller/hooks/hook-pyopencl.py

```diff
@@ -0,0 +1,13 @@
+#-----------------------------------------------------------------------------
+# Copyright (c) 2015-2016, PyInstaller Development Team.
+#
+# Distributed under the terms of the GNU General Public License with exception
+# for distributing bootloader.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+#-----------------------------------------------------------------------------
+
+# Hook for the pyopencl module: https://github.com/pyopencl/pyopencl
+
+from PyInstaller.utils.hooks import copy_metadata
+datas = copy_metadata('pyopencl')
```

Comment:
Now I am a bit confused, #2305 has been closed as a duplicate of this PR. I have come across #2305 randomly while looking for some other pyinstaller related issues.